WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift

```diff
@@ -88,11 +98,18 @@ final class QuickStartPromptViewController: UIViewController {
     // MARK: - Setup
 
     private func setup() {
+        setupScrollViewMargins()
         setupSiteInfoViews()
         setupPromptInfoViews()
         setupButtons()
     }
 
+    private func setupScrollViewMargins() {
+        let margin = view.getHorizontalMargin() + 20.0
```

Comment:
A pattern that is somewhat common in this project is to extract numbers, strings, etc to an `enum Constants`.

Just search for `enum Constants` and you'll see a bunch of examples. Wdyt about extracting that to a enum?

Reply:
This value would go away if I use the `readableContentGuide` like you suggested.